Sources/Plasma/PubUtilLib/plAnimation/CMakeLists.txt

```diff
@@ -27,11 +27,25 @@ set(plAnimation_HEADERS
 )
 
 add_library(plAnimation STATIC ${plAnimation_SOURCES} ${plAnimation_HEADERS})
-
-target_link_libraries(plAnimation pnModifier)
-target_link_libraries(plAnimation plInterp)
-target_link_libraries(plAnimation plSDL)
-target_link_libraries(plAnimation plTransform)
+target_link_libraries(
+    plAnimation
+    PUBLIC
+        CoreLib
+        pnFactory
+        pnModifier
+        pnNetCommon
+        plTransform
+        string_theory
```

Comment:
`string_theory` already linked by `CoreLib`

... and here is where I stop for now ...  >.>